src/settings/FinePolicy/FinePolicyDetail.test.js

```diff
@@ -0,0 +1,285 @@
+        accordionStatus: accordionDefaultStatus,
+      }), {});
+
+      expect(Accordion).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          open: accordionDefaultStatus.generalFeePolicy,
+        }), {}
+      );
+
+      expect(FinesSection).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          fineSectionOpen: accordionDefaultStatus.viewFineSection,
+        }), {}
+      );
+
+      fireEvent.click(screen.getByTestId('expandAllButton'));
+
+      expect(ExpandAllButton).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: {
+          generalFeePolicy: false,
+          viewFineSection: false,
+        },
+      }), {});
+
+      expect(AccordionSet).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: {
+          generalFeePolicy: false,
+          viewFineSection: false,
+        },
+      }), {});
+
+      expect(Accordion).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          open: false,
+        }), {}
+      );
+
+      expect(FinesSection).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          fineSectionOpen: false,
+        }), {}
+      );
+    });
+  });
+
+  describe('handleSectionToggle method', () => {
+    it('should extapnd accordion', () => {
+      render(
+        <FinePolicyDetail {...testDefaultProps} />
+      );
+
+      expect(ExpandAllButton).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: accordionDefaultStatus,
+      }), {});
+
+      expect(AccordionSet).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: accordionDefaultStatus,
+      }), {});
+
+      expect(Accordion).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          open: accordionDefaultStatus.generalFeePolicy,
+        }), {}
+      );
+
+      fireEvent.click(screen.getByTestId('accordionSet'));
+
+      expect(ExpandAllButton).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: {
+          ...accordionDefaultStatus,
+          generalFeePolicy: false,
+        },
+      }), {});
+
+      expect(AccordionSet).toHaveBeenLastCalledWith(expect.objectContaining({
+        accordionStatus: {
+          ...accordionDefaultStatus,
+          generalFeePolicy: false,
+        },
+      }), {});
+
+      expect(Accordion).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          open: false,
+        }), {}
+      );
+    });
+  });
+
+  describe('getCheckboxValue method', () => {
+    it('should get checkbox value when seletedValue is true', () => {
+      FinesSection.mockImplementationOnce(({ getCheckboxValue }) => (
+        <div data-testid="finesSection">
+          {getCheckboxValue(true)}
+        </div>
+      ));
+
+      render(
+        <FinePolicyDetail {...testDefaultProps} />
+      );
+
+      expect(getById('finesSection').getByText(labelIds.finePolicyYes)).toBeVisible();
```

Comment:
I think it's better to move all ids into the one object

Reply:
done